src/server/game/AI/SmartScripts/SmartScriptMgr.cpp

```diff
@@ -868,21 +868,6 @@ bool SmartAIMgr::IsEventValid(SmartScriptHolder& e)
                     TC_LOG_ERROR("sql.sql", "SmartAIMgr: Entry %d SourceType %u Event %u Action %u has pct value above 100, skipped.", e.entryOrGuid, e.GetScriptType(), e.event_id, e.GetActionType());
                     return false;
                 }
-
-                switch (e.GetTargetType())
```

Comment:
could you describe a bit more in detail why this was removed ?

Reply:
It's been a very long time since I did this so will have to look over it properly again. Off the top of my head though the event is just a broken hack, and it's trying to force specific targets (like range), when none of these actually work since they'll just get a new list of targets and everything originally checked for health % is lost.